sig-usability/annual-report-2021.md

```diff
@@ -0,0 +1,53 @@
+# 2021 Annual Report: SIG Usability
+
+## Operational
+
+How are you doing with operational tasks in sig-governance.md?
+Is your README accurate? have a CONTRIBUTING.md file?
+Yes, updated to reflect new meeting time
+https://github.com/kubernetes-sigs/sig-usability 
+
+All subprojects correctly mapped and listed in sigs.yaml?
+N/A
+
+What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+- Videos are all here: https://www.youtube.com/watch?v=cujQhg6JcgI&list=PL69nYSiGNLP0eY-U8DVJWHBwKvMDEtOxx 
+- We have small, interactive meetings, usually discuss and coordinate around our user research study that’s on-going 
+
+How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+N/A
+
+Same question as above but for working groups.
+N/A
+
+When was your last monthly community-wide update? (provide link to deck and/or recording)
+I do not recall 
+
+## Membership
+
+Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+We need to finalize the PR on kubernetes/kubernetes to make Gaby formally a chair 
+
+How do you measure membership? By mailing list members, OWNERs, or something else?
+We engage with membership via Slack and mailing list 
```

Comment:
do you consider members of the group to be active participants on these channels? do work in subproject? how is that defined?

Reply:
We don't have any formal sub-projects. Any one is welcome to participate. They can join the mailing list to get updates, or be in the slack channel, or attend meetings.